modules/UI/UI.js

```diff
@@ -1037,25 +1037,25 @@ UI.requestFeedbackOnHangup = function () {
     if (Feedback.isVisible())
         return Promise.reject(UIErrors.FEEDBACK_REQUEST_IN_PROGRESS);
     // Feedback has been submitted already.
-    else if (Feedback.isEnabled() && Feedback.isSubmitted())
-        return Promise.resolve(true);
+    else if (Feedback.isEnabled() && Feedback.isSubmitted()) {
+        return Promise.resolve({
+            showThankYou : true,
+            feedbackSubmitted: true
```

Comment:
If our html-s are called close and close2, let's rename the variable to closePageVisible instead of showThankYou?